src/import/FSHImporter.ts

```diff
@@ -1499,6 +1499,9 @@ export class FSHImporter extends FSHVisitor {
     const str = stringCtx.getText();
     return str
       .slice(1, str.length - 1)
+      .replace(/\\n/g, '\n')
+      .replace(/\\r/g, '\r')
+      .replace(/\\t/g, '\t')
```

Comment:
These replacements need to happen after the literal-backslash replacement. Otherwise, this FSH input would cause problems: `Some \\n text`

Reply:
Good catch! I updated this and added a test to check that `Some \\n text` will get replaced properly and result in `Some \n text`.